src/config.ts

```diff
@@ -84,6 +78,7 @@ export interface LabelTriageWorkflowConfig {
   repoName: string
   repository: string
   issueNumber: number
```

Comment:
Should this not also be nullable since we could be only using queries? When we set, we probably should not fall back to 0 but let it stay undefined.

Then we fix up the code to skip or throw or something based on null. A 0 issue may be unintentional.